docs/source/faq/faq.md

```diff
@@ -1,7 +1,7 @@
 # Frequently asked questions
 
 The following lists a set of questions that we have been asked about Kedro in the past. If you have a different
- question which isn't answered here, check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Discord Server](https://discord.gg/akJDeVaxnB).
+ question which isn't answered here, check out [GitHub Discussions](https://github.com/kedro-org/kedro/discussions) or talk to the community on the [Slack organisation](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw).
```

Comment:
```suggestion
 question which isn't answered here, check out the [searchable archive from our retired Discord server](https://linen-discord.kedro.org/community or post a new query on the [Slack organisation](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw).
```

Reply:
Here, the link to the linen archive above seems to return a 404. Maybe you meant [https://linen-discord.kedro.org](https://linen-discord.kedro.org) instead?

Reply:
I did, thanks!

Reply:
@BSGalvan I've now fixed that link throughout my suggestions where it arose, so all should work as expected.